docs/reference/installation/_interfaces/helm.cue

```diff
@@ -0,0 +1,128 @@
+package metadata
+
+installation: _interfaces: helm: {
+	title:       "Helm"
+	description: """
+		[Helm](\(urls.helm)) is a package manager for Kubernetes that
+		facilitates the deployment and management of services on Kubernetes
+		clusters.
+		"""
+
+	archs: ["x86_64", "ARM64"]
+	paths: {
+		bin:         "/usr/bin/vector"
+		bin_in_path: true
+		config:      "configs/vector.{config_format}"
+	}
+	package_manager_name: installation.package_managers.helm.name
+	platform_name:        installation.platforms.kubernetes.name
+	roles: {
+		_commands: {
+			_name:          string
+			_resource_type: string
+			configure:      #"""
+						cat <<-VECTORCFG > \#(paths.config)
+						{config}
+						VECTORCFG
+						"""#
+			install:        #"""
+				 helm repo add timberio-nightly https://packages.timber.io/helm/nightly && \
```

Comment:
AFAICT the Helm instructions should call for `helm repo update` after `helm repo add`. There may be a way around that but I've checked and haven't found one.

Reply:
In our case it is not required, as we don't have any external dependencies.
Also, with Helm 3, a repo update is issued on the initial repo addition automatically.